### PR TITLE
removing unused import causing circular dependency

### PR DESCRIPTION
**Pull request by ferrigno**
This simple patch seems to solve a circular import issue

### CODE REVIEW

**Review by dsavchenko — APPROVED** [review]
Thank you!

I really wonder why we didn't see the issue before

File: dispatcher_plugin_nb2workflow/queries.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
                        NB2WTextProduct,
                        NB2WParameterProduct,
                        NB2WProgressProduct)
-from .dataserver_dispatcher import NB2WDataDispatcher
+#from .dataserver_dispatcher import NB2WDataDispatcher
```
Comment (volodymyrss) [inline dispatcher_plugin_nb2workflow/queries.py:10]: There is no point in keeping commented code.

```suggestion
```

**Review by volodymyrss — APPROVED** [review]
Seems like this import is not used, strange that it's there. Good to remove, thanks!

Note @ferrigno that I renamed the PR to avoid just reciting the relation with issue which is already tracked by github. 
Also clear title will help to understand the change in the future.

**Comment by dsavchenko** [comment]
> strange that it's there

It's there from the initial implementation, which was largely refactored then

**Pull request merged**